conf/supervisord-dev.conf

```diff
@@ -0,0 +1,50 @@
+[supervisord]
+nodaemon = true
+
+[program:web]
+command = tox -qqe dev --run-command 'pserve --reload conf/development-app.ini'
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
+killasgroup = true
+
+[program:websocket]
+command = tox -qqe dev --run-command 'pserve --reload conf/development-websocket.ini'
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
+killasgroup = true
+
+[program:worker]
+command = tox -qqe dev --run-command 'sh bin/hypothesis --dev celery worker'
```

Comment:
We could also try to get rid of `bin/hypothesis celery ...` and just run Celery directly, but unlike `devserver` `bin/hypothesis celery ...` is actually used in production, so I'd leave fiddling with that to a separate PR if at all. (This PR doesn't affect prod at all.)